ffmpeg/lpms_ffmpeg.c

```diff
@@ -58,6 +72,22 @@ struct output_ctx {
   int64_t drop_ts;     // preroll audio ts to drop
 
   output_results  *res; // data to return for this output
+
+  int nb_frame; // XXX temp
```

Comment:
Why is this marked as temp?

Reply:
Ah, because it might not be needed after all. (This would also imply that the `force-idr` AVOption isn't required either.) Checking and will follow up.

Reply:
Keeping track of the frame count is still necessary, but we already have that information for the other pixel counts, so we rely on `res.frames` rather than using this separate counter. Removed `nb_frame` from the struct.